package.json

```diff
@@ -5,7 +5,7 @@
   "bin": "bin.js",
   "main": "index.js",
   "dependencies": {
-    "ci-info": "^1.3.0"
+    "ci-info": "^1.*"
```

Comment:
I prefer just leaving this dependency version number as it is. It will cover the same range except not `<1.3.0`, but I don't see any reason to include those old versions - unless you have a specific reason?

Reply:
I agree, but when you do this. This may satisfy `ci-info v2`, which may introduce any breaking change. Projects use this lib may break.  Hence `1.*` will be suitable, Once You release `ci-info v2`, You can manually release a new version of `is-ci` referring `2.*`

Reply:
Apart from this. what you suggested will work too. there is no big difference.

Reply:
`^1.3.0` expands to `>=1.3.0 <2.0.0`, so it will not match the next major. You can read more about caret (^) ranges here: https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Reply:
Yes, Once a lock file is generated and doing `npm install` the behaviours will differ I guess, let me check once.

Reply:
>  but I don't see any reason to include those old versions - unless you have a specific reason?

It doesn't mean including old versions, just any. Whatever the latest is.

Reply:
How will `1.*` differ from `^1.0.0` when there's a lock file?

Reply:
Nope, I still use npm 5, what I thought of happening there was a bug I guess. Seems it got fixed after npm v5.4.2. Funny I thought it was a feature 😅

Reply:
`1.*` and `^1.0.0` are same, I had a misconception about that 😓 

But since `*` represents. You can keep it if you want, as `is-ci` works with any version of `ci-info`. when installing it always points the latest version available on npm registry.

Reply:
I think I'd like to keep it as `^1.3.0` mainly to ensure that bugs fixed in older versions of ci-info isn't re-introduced if someone have another dependency to an older version of ci-info or something like that. I know it's a very remote risk, but as there's no benefit of downgrading the minimum required version, I don't think it's worth it 😃

Reply:
> I know it's a very remote risk.

Even I became unsure now 😅 .